ra2ce/graph/networks.py

```diff
@@ -146,7 +147,7 @@ def _export_network_files(
             output_dir=self.output_graph_dir,
             export_types=types_to_export,
         )
-        self.files[graph_name] = _exporter.get_pickle_path()
+        setattr(self.files, graph_name, _exporter.get_pickle_path())
```

Comment:
Avoid using `setattr`, please be explicit. Now that you have an object you can defer the setting of this attribute in a method within the `GraphFiles` such as `GraphFiles.set_files`, in there you can map whatever `graph_name` is given to the corresponding field and even raise an error if it does not exist. If I'm not mistaken your current approach will even set a new attribute when it does not exist.

Reply:
Done